### ISSUE TITLE
CSS not being compressed

### ISSUE THREAD

**Issue by adamhl8** [nhristov/astro-compress#422]
I'm not able to compress the CSS generated by astro.

```
Error: Cannot compress file //dist/_astro/_slug_.CPIc14WV.css
Error: Cannot compress file //dist/_astro/_slug_.Dvd5riTk.css
```

In #376 it's mentioned that if a file doesn't need compression an error is thrown, but I don't think that's what's happening in my case.

I manually installed `lightningcss` and ran it on one of the files and it went from 52KB to 48KB, so it can definitely be compressed.

I thought it might be something with `csso` specifically, so I tried setting the parser to just `lightningcss`, but I still get the same error.

```
playformCompress({ Parser: { CSS: "lightningcss" } }),
```

If you'd like to reproduce, it's on the branch `compress` on my repo: https://github.com/adamhl8/adamhl.dev

**Comment by geepot** [comment]
Running into this same issue in my environment

**Comment by NikolaRHristov** [comment]
We found the error.

We're setting `unusedSymbols` to `false` in https://github.com/PlayForm/Compress/blob/c807585b8525baccfe6e40d052c684b833c91f10/Source/Variable/CSS/lightningcss.ts#L10 and it's producing the error:

```sh
Error: invalid type: boolean `false`, expected a sequence
```

As a hotfix I would recommend setting `unusedSymbols` to `[]`:

```typescript
import type { defineConfig } from "astro/config";

export default (await import("astro/config")).defineConfig({
	integrations: [
		(await import("@playform/compress")).default({
			CSS: {
				lightningcss: {
					unusedSymbols: [],
				},
			},
		}),
	],
}) as typeof defineConfig;
```

**Comment by NikolaRHristov** [comment]
Fixed in the latest [@playform/compress@v0.1.5](https://github.com/PlayForm/Compress/releases/tag/Compress%2Fv0.1.5) and [astro-compress@v2.3.4](https://github.com/PlayForm/Compress/releases/tag/AstroCompress%2Fv2.3.4)
